src/main/java/io/github/nucleuspowered/nucleus/modules/core/commands/itemalias/SetItemAliasCommand.java

```diff
@@ -68,8 +69,8 @@ public CommandResult executeCommand(CommandSource src, CommandContext args) thro
             // If player, get the item in hand, otherwise, we can't continue.
             if (src instanceof Player) {
                 Player pl = (Player)src;
-                if (pl.getItemInHand().isPresent()) {
-                    final ItemStack is = pl.getItemInHand().get();
+                if (pl.getItemInHand(HandTypes.MAIN_HAND).isPresent()) {
```

Comment:
Why? I _could_ do that, but it isn't the _right_ thing to do. I think it's unexpected, unnecessary behaviour, someone might have misscrolled onto an empty slot and still affect an item.

Let's not over complicate the interaction between Nucleus and the user for the sake of.